youtube_dl/extractor/pornoxo.py

```diff
@@ -26,22 +28,16 @@ class PornoXOIE(InfoExtractor):
 
     def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
-        video_id = mobj.group('id')
+        video_id, display_id = mobj.groups()
 
         webpage = self._download_webpage(url, video_id)
 
-        video_url = self._html_search_regex(
-            r'\'file\'\s*:\s*"([^"]+)"', webpage, 'video_url')
+        video_data = self._parse_json(self._search_regex(
+            r'(?s)jwplayer\("mediaplayer"\)\.setup\(({[^;]+})\);',
+            webpage, 'video data'), video_id, transform_source=js_to_json)
 
-        title = self._html_search_regex(
-            r'<title>([^<]+)\s*-\s*PornoXO', webpage, 'title')
-
-        description = self._html_search_regex(
-            r'<meta name="description" content="([^"]+)\s*featuring',
-            webpage, 'description', fatal=False)
-
-        thumbnail = self._html_search_regex(
-            r'\'image\'\s*:\s*"([^"]+)"', webpage, 'thumbnail', fatal=False)
+        video_url = video_data['file']
+        title = self._html_search_regex(r'<h1>([^<]+)</h1>', webpage, 'title')
```

Comment:
No, but `<title>` contains more than the title. Should I rely on the old method, and have this as a fallback?